app/serializers/api/v2/chapters/chapter_serializer.rb

```diff
@@ -13,7 +13,8 @@ class ChapterSerializer
                    :description,
                    :formatted_description,
                    :validity_start_date,
-                   :validity_end_date
+                   :validity_end_date,
+                   :description_plain
```

Comment:
How do you feel about bringing the plain description next to the formatted description?